ceremonies/src/tests.rs

```diff
@@ -1449,13 +1453,13 @@ fn grow_population_works() {
 }
 
 #[rstest(n_bootstrappers, n_reputables, n_endorsees, n_newbies, n_locations, exp_meetups,
-    case(8,0,0,4,3, vec![12]),
-    case(9,0,0,4,3, vec![7,6]),
-    case(3,7,3,3,3, vec![8,8]),
-    case(3,7,4,3,3, vec![9,8]),
-    case::do_not_assign_more_meetups_than_locations(3,7,50,0,3, vec![12,12,12]),
-    case::do_not_assign_more_meetups_than_there_are_experienced_participants(3,1,49,0,10, vec![12,12,12,12]),
-    case(12,48,12*AMOUNT_NEWBIE_TICKETS as usize,0,55, [12; 55].to_vec()),
+case(8, 0, 0, 4, 3, vec ! [12]),
```

Comment:
this won't compile, right? whitespace in `vec!`? why?

Reply:
did not have this checked, have it now. thank you